docs/analysis-test-add-sh-to-path (#1166)
MaibornWolff/codecharta: gh-pages/_docs/07-01-new-to-code.md

-   Run `gradlew.bat test` or `./gradlew test`
-   Run `gradlew.bat integrationTest` or `./gradlew integrationTest`

> May the integration tests will fail because of missing or unknown `sh` command.
> To make it work you could add the path to the Git `sh.exe` (which is normally placed here `C:\<path-to-git>\Git\bin`) to your PATH variable.

Reviewer: ```suggestion
The integration tests might fail on windows, because of a missing or unknown `sh` command.
To make it work, add the path to the Git `sh.exe` (which is normally placed here `C:\<path-to-git>\Git\bin`) to your PATH variable.
```
